src/scripts/main.js

```diff
@@ -1,3 +1,46 @@
 'use strict';
 
-// write code here
+const liElements = document.querySelectorAll('li');
+const ulElement = document.querySelector('ul');
+const arr = [...liElements];
+
+function toNumb(salary) {
```

Comment:
```suggestion
function convertToNumber(salary) {
```
Let's get used to name function according to these rules: 
![image](https://user-images.githubusercontent.com/96409563/208856651-77d6feff-c420-4f2a-ae27-901197f38730.png)